### DISCUSSION TITLE
guide/components/swiper/

### DISCUSSION THREAD

**Discussion by giscus** [pengzhanbo/vuepress-theme-plume#172] [docs-comment]
# guide/components/swiper/

概述 使用 <Swiper> 组件在 页面中显示轮播图片。 使用 使用该组件，首先需要手动安装 swiper 库： 然后，手动导入 Swiper 组件： 注册为全局组件： 全局组件可在 其他任意 markdown 文件中使用 示例:

https://theme-plume.vuejs.press/guide/components/swiper/

**Comment by cjhalw** [comment]
:100:

**Comment by sunshang-hl** [comment]
cards这种轮播图是否能让图片循环展示?目前这种展示到最后或者第一张图片时,左右两边显示内容不对称.

**Reply by pengzhanbo** [reply]
相关需求请到 [swiper](https://swiperjs.com/) 提出，主题通过它实现的 轮播图。

**Comment by rubinoxyl** [comment]
我通过添加 width=80%将轮播图宽度设为80%，但是图片宽度缩小后是居左的，右边有一块空白，

**Reply by pengzhanbo** [reply]
在 issue 中贴一下你的具体代码

**Reply by rubinoxyl** [reply]
下面是代码，把width设置为了80%
<img width="837" height="214" alt="image" src="https://github.com/user-attachments/assets/f6581705-ac64-4dea-9172-db177abae608" />
最后轮播图没有居中，右边有空白
<img width="1387" height="669" alt="Snipaste_2025-12-26_11-54-59" src="https://github.com/user-attachments/assets/27851c4d-b000-4327-bde7-dfa4ee867512" />

**Reply by pengzhanbo** [reply]
将 Swiper 放到 `::: felx center` 容器中进行居中即可。<https://theme-plume.vuejs.press/guide/markdown/flex/>
